Source/WarpX.H

```diff
@@ -69,19 +69,45 @@
 #include <array>
 #include <iostream>
 #include <limits>
+#include <map>
 #include <memory>
 #include <optional>
 #include <string>
 #include <vector>
-#include <map>
-
 
 enum struct PatchType : int
 {
     fine,
     coarse
 };
 
+enum struct FieldType : int
```

Comment:
This is ok for now, but we should make it a runtime construct so we can add more multifabs later on at runtime and also easily name them.